src/helpers/context.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import JubilationTheme from '../jubilation-theme';
 
-export default function getContext(context: JubilationContext) : JubilationContext {
+export default function getContext(context: JubilationContext): JubilationContext {
```

Comment:
yeah, this is a little bit of magic. we register a set of type definitions in our `.flowconfig` in https://github.com/coreyflynn/jubilation/blob/master/.flowconfig#L8

from there, flow picks up the declared types in our `src/types/index`. Specifically, `JubilationContext` is declared in https://github.com/coreyflynn/jubilation/blob/master/src/types/index.js#L23

doing that makes it globally available to flow